addons/binding/org.openhab.binding.fmiweather.test/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I see a number of files that should be removed and the .classpath and .project in bundle don't seem ok either.

Reply:
Thank you, it feels very good that someone is finally reviewing the code. 

I guess this all was bad timing on my part, and whole build setup has been quite in flux. I have been trying to incorporate the new conventions, though.

What would be the most sensible way to proceed? could you please give some more pointers to the issue?

Thanks

Reply:
I wrote a tutorial on the forum on how to migrate a binding. I guess the easiest way to proceed is to follow those steps(or take from it what can help you). And instead of coping from the old location, copy the files from `src/` and readme related files from your new location. See: https://community.openhab.org/t/tutorial-migrate-your-binding-to-the-maven-bnd-based-build-system/81389

Reply:
Thanks, I've noticed the tutorial, and I thought I executed the steps in this repo as well when converting it to newstyle some months back.

But obviously it looks like I have skipped some steps or something like that.

Will try again and let you know then.